libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -255,7 +253,7 @@ InternalDispatchMap IRGenerator::generateInternalDispatchFunctions()
 	);
 
 	InternalDispatchMap internalDispatchMap = m_context.consumeInternalDispatchMap();
-	for (YulArity const& arity: internalDispatchMap | boost::adaptors::map_keys)
+	for (YulArity const& arity: ranges::views::keys(internalDispatchMap))
```

Comment:
This file even already contains `using namespace ranges;`:
```suggestion
	for (YulArity const& arity: views::keys(internalDispatchMap))
```

Reply:
Not changed as discussed above to use explicit ranges::